code/game/gamemodes/gang/dominator.dm

```diff
@@ -13,6 +13,7 @@
 	var/datum/gang/gang
 	var/operating = 0	//0=standby or broken, 1=takeover
 	var/warned = 0	//if this device has set off the warning at <3 minutes yet
+	var/spam_prevention = 100
```

Comment:
if you want that then at the start of this file:

```#define DOM_BLOCKED_SPAM_CAP 6``` (because it is 6 atm)
swap this line for ```var/spam_prevention = DOM_BLOCKED_SPAM_CAP //first message is immediate```
swap ```if(spam_prevention < 6)``` for ```if(spam_prevention < DOM_BLOCKED_SPAM_CAP)```
and at the end of the file, ```#undef DOM_BLOCKED_SPAM_CAP```